### PR TITLE
net/frr: added as-override to gui

### PR DESCRIPTION
**Pull request by winglessza**
This is to bring in the ability add the as-override command to the neighbor peers so that the AS number of learned local peers can be overridden to the local AS number. This only works with EBGP peers but is necessary to use when wanting to filter out VRF based peering traffic and advertising learned local routes into your ebgp network space.

### CODE REVIEW

File: net/frr/src/opnsense/service/templates/OPNsense/Quagga/bgpd.conf [hunk 1]
```diff
@@ -212,4 +215,4 @@ agentx
 !
 line vty
 !
-{% endif %}
+{% endif %}
```
Comment (fichtner) [inline net/frr/src/opnsense/service/templates/OPNsense/Quagga/bgpd.conf:218]:
```suggestion
{% endif %}

```

**Comment by fichtner** [comment]
@mimugmail ping

File: net/frr/src/opnsense/mvc/app/controllers/OPNsense/Quagga/forms/dialogEditBGPNeighbor.xml [hunk 2]
```diff
@@ -89,6 +89,12 @@
     <label>Send Defaultroute</label>
     <type>checkbox</type>
   </field>
+  <field>
+    <id>neighbor.asoverride</id>
+    <label>Enable As-Override</label>
```
Comment (mimugmail) [inline net/frr/src/opnsense/mvc/app/controllers/OPNsense/Quagga/forms/dialogEditBGPNeighbor.xml:94]: AS-Override :)

**Review by mimugmail — APPROVED** [review]
Looks good, thx :)

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**